lib/honeybadger/agent.rb

```diff
@@ -143,6 +143,25 @@ def notify(exception_or_opts, opts = {})
       notice.id
     end
 
+    # Public: Perform a synchronous check_in.
+    #
+    # id - The unique check in id (e.g. '1MqIo1') or the check in url
+    #      (e.g. 'https://api.honeybadger.io/v1/check_in/1MqIo1').
+    #
+    # Examples
+    #
+    #
+    #   Honeybadger.check_in('1MqIo1')
+    #   Honeybadger.check_in('https://api.honeybadger.io/v1/check_in/1MqIo1')
```

Comment:
Since we want to prefer passing just the check-in id, let's remove this line from the examples so that we don't encourage its use.